### DISCUSSION TITLE
is it possible to take a snapshot of the actual flight and load after?

### DISCUSSION THREAD

**Discussion by victordeveloperbr** [flybywiresim/aircraft#8237] [Ideas]
Hello, I would like to request a feature if possible, I'm a real pilot and I would like to use a fenix for my personal training but I'm stuck in a question, there will be any possibility to save a present snapshot of the flight where it will include mcdu settings like flight plan, performance, etc and after we can load it snapshot? because each time when we load the aircraft we have to insert all again and it loose a lot of time for training, since now thank you for attention.

**Comment by ExampleWasTaken** [comment]
Not sure if that's just a little oversight but this is the FlyByWire Simulations project so A32NX, not FENIX.
